lib/geocoder/lookups/here_calculate_route.rb

```diff
@@ -26,15 +26,12 @@ def results(query)
     end
 
     def query_url_params(query)
-      super.merge(query_url_here_options(query)).merge(
-        waypoints_hash(query.to_param)
-      )
+      super.merge(query_url_here_options(query))
     end
 
-    def waypoints_hash(waypoints)
-      waypoints.each_with_object({}).with_index do |(point, hash), index|
-        hash["waypoint#{index}"] = point.join(',')
-      end
+    def hash_to_query(hash)
+      URI.encode_www_form(hash)
```

Comment:
How is this ensuring waypoint are converted to format specified here:
https://developer.here.com/documentation/routing-api/dev_guide/topics/use-cases/via.html


We use this in HereMaps::ApiHelper#get_transport_report